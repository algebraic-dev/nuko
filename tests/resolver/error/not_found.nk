let a (b: Int) (c: Int) = d