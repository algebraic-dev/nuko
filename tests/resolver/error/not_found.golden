This CannotFindInModule
    ( VarName :| [] ) Nothing "d"
    ( Range
        { start = Pos
            { line = 0
            , column = 26
            }
        , end = Pos
            { line = 0
            , column = 27
            }
        }
    )
